<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QWidget" name="widget_8" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>167</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>最小值</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sMin"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>最大值</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sMax">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>数字类型</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QRadioButton" name="sIsInt">
                  <property name="text">
                   <string>整数</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="sIsFloat">
                  <property name="text">
                   <string>小数</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>行尾处理</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QRadioButton" name="sIsNextLine">
                  <property name="text">
                   <string>换行</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="sIsSpace">
                  <property name="text">
                   <string>空格</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_10" native="true">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>小数位数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="sNumOfDecimal"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sBtnAdd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>添加单个随机数</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>249</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>221</width>
            <height>45</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>最小值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="aMin"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>30</y>
            <width>211</width>
            <height>45</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>最大值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="aMax"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>221</width>
            <height>45</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>最小长度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="aMinLen"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_9" native="true">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>100</y>
            <width>211</width>
            <height>45</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>最大长度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="aMaxLen"/>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="aBtnAdd">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>931</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>添加随机数组</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>40</y>
            <width>141</width>
            <height>86</height>
           </rect>
          </property>
          <property name="title">
           <string>数字类型</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QRadioButton" name="aIsInt">
             <property name="text">
              <string>整数</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="aIsFloat">
             <property name="text">
              <string>小数</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>40</y>
            <width>141</width>
            <height>86</height>
           </rect>
          </property>
          <property name="title">
           <string>元素分隔处理</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QRadioButton" name="aIsNextLine">
             <property name="text">
              <string>换行</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="aIsSpace">
             <property name="text">
              <string>空格</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QCheckBox" name="addAmout">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>150</y>
            <width>161</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>在最初添加数组长度</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="widget_11" native="true">
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>50</y>
            <width>200</width>
            <height>86</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>小数位数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="aNumOfDecimal"/>
           </item>
          </layout>
         </widget>
         <widget class="QCheckBox" name="aAoumntNextLine">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>150</y>
            <width>161</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>数组长度换行</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>生成文件数量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="amount">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGenerate">
            <property name="text">
             <string>生成文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReset">
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
